### ISSUE TITLE
hgboost installation error 

### ISSUE THREAD

**Issue by chandrabhuma** [erdogant/hgboost#18]
When installing in colab...this error is coming 
error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  Preparing metadata (setup.py) ... error
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

Could you pl help

**Comment by erdogant** [comment]
Both [colab notebooks](https://erdogant.github.io/hgboost/pages/html/Documentation.html#colab-classification-notebook) are working on my machine. Do you pip install the package? or install it from github source?

**Comment by chandrabhuma** [comment]
I checked once again in COLAB PRO. I have installed using PIP. The error message is like this...Some times it is working but most of the times it is showing the following error
Collecting hgboost
  Using cached hgboost-1.1.5-py3-none-any.whl (28 kB)
Collecting datazets (from hgboost)
  Using cached datazets-0.1.9-py3-none-any.whl (51 kB)
Collecting pypickle (from hgboost)
  Using cached pypickle-1.1.0-py3-none-any.whl (5.1 kB)
Requirement already satisfied: matplotlib in /usr/local/lib/python3.10/dist-packages (from hgboost) (3.7.1)
Requirement already satisfied: numpy in /usr/local/lib/python3.10/dist-packages (from hgboost) (1.23.5)
Requirement already satisfied: pandas in /usr/local/lib/python3.10/dist-packages (from hgboost) (1.5.3)
Requirement already satisfied: tqdm in /usr/local/lib/python3.10/dist-packages (from hgboost) (4.66.1)
Requirement already satisfied: hyperopt in /usr/local/lib/python3.10/dist-packages (from hgboost) (0.2.7)
Requirement already satisfied: lightgbm>=4.1.0 in /usr/local/lib/python3.10/dist-packages (from hgboost) (4.1.0)
Collecting catboost (from hgboost)
  Using cached catboost-1.2.2-cp310-cp310-manylinux2014_x86_64.whl (98.7 MB)
Requirement already satisfied: xgboost in /usr/local/lib/python3.10/dist-packages (from hgboost) (2.0.1)
Collecting classeval (from hgboost)
  Using cached classeval-0.2.1-py3-none-any.whl (16 kB)
Collecting treeplot (from hgboost)
  Using cached treeplot-0.1.16-py3-none-any.whl (8.7 kB)
Collecting df2onehot (from hgboost)
  Using cached df2onehot-1.0.6-py3-none-any.whl (14 kB)
Collecting colourmap (from hgboost)
  Using cached colourmap-1.1.16-py3-none-any.whl (10 kB)
Requirement already satisfied: seaborn in /usr/local/lib/python3.10/dist-packages (from hgboost) (0.12.2)
Requirement already satisfied: scipy in /usr/local/lib/python3.10/dist-packages (from lightgbm>=4.1.0->hgboost) (1.11.3)
Requirement already satisfied: graphviz in /usr/local/lib/python3.10/dist-packages (from catboost->hgboost) (0.20.1)
Requirement already satisfied: plotly in /usr/local/lib/python3.10/dist-packages (from catboost->hgboost) (5.15.0)
Requirement already satisfied: six in /usr/local/lib/python3.10/dist-packages (from catboost->hgboost) (1.16.0)
Requirement already satisfied: python-dateutil>=2.8.1 in /usr/local/lib/python3.10/dist-packages (from pandas->hgboost) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.10/dist-packages (from pandas->hgboost) (2023.3.post1)
Collecting funcsigs (from classeval->hgboost)
  Using cached funcsigs-1.0.2-py2.py3-none-any.whl (17 kB)
Collecting sklearn (from classeval->hgboost)
  Using cached sklearn-0.0.post10.tar.gz (3.6 kB)
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  Preparing metadata (setup.py) ... error
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
